setup.py

```diff
@@ -31,7 +31,8 @@
     ],
     include_package_data=True,
     install_requires=[
-        'Django>=1.10',
+        'Django==1.11,<2.1;python_version<"3.5"',
+        'Django>=1.11,<2.2;python_version>"3.4"',
```

Comment:
@Eorhim do I read it correctly that 

* if 1.11 or 2.0, then python must be less than 3.5
* if 1.11, 2.0, or 2.1 then python must be greater than 3.4

that seems like double definitions for me, which might even be contradictory
I would probably go with a line per python version
```suggestion
        'Django==1.11;python_version=="2.7"',
        'Django>=1.11,<2.1;python_version=="3.4"',
        'Django>=1.11,<2.2;python_version>="3.5",
```
however, as this whole thing seems like a [minefield](https://hynek.me/articles/conditional-python-dependencies/) maybe it would be better to drop it altogether, and just assume that the [`Django` installation took care of this](https://github.com/django/django/blob/master/setup.py#L72)?
```suggestion
        'Django>=1.11,<2.2',
```